src/m_menu.c

```diff
@@ -4021,66 +4069,92 @@ setup_menu_t gen_settings1[] = { // General Settings screen1
   {0,S_SKIP|S_END,m_null}
 };
 
-// Page 2
+setup_menu_t gen_settings2[] = { // General Settings screen2
 
-enum {
-  gen2_title1,
-  gen2_smooth_scaling,
-  gen2_trans,
-  gen2_transpct,
-  gen2_stub1,
-  gen2_sky1,
-  gen2_sky2,
-  gen2_swirl,
-  gen2_smoothlight,
-  gen2_brightmaps,
-  gen2_stub2,
-  gen2_solidbackground,
-  gen2_menu_background,
-  gen2_diskicon,
-  gen2_endoom,
-  gen2_end1,
+  {"Advanced Sound", S_SKIP|S_TITLE, m_null, M_X, M_Y},
```

Comment:
I am still a bit unsure how to knit this into the existing menu structure. Ideally, I would like to have all sound and music options on one page. We could move the current sound options from page 1 to page 2, but then page 1 would be a bit empty. Maybe move translucency settings back from the "Display Options" page? @rfomin Any better idea? If not, we can also post-pone this to a later point and focus on the technical implementation first.

Reply:
I agree, and yes, we should postpone the menu rearrangement when all features are complete.